tensorboard/plugins/profile/profile_plugin.py

```diff
@@ -158,6 +159,8 @@ def data_impl(self, run, tool):
       return None
     if tool == 'trace_viewer':
       return process_raw_trace(raw_data)
+    if tool == 'op_profile':
+      return raw_data
```

Comment:
We should definitely discuss whether the profile data can be retrieved through the graph as a TF summary. There are some weird requirements/nice-to-haves here, but maybe also a little fear-of-the-unknown from our side :-)

I've removed the comment because the action it suggests isn't the right one.